app/models/history.rb

```diff
@@ -32,7 +32,8 @@ class History < ApplicationRecord
     contact_attempt: 'Contact Attempt',
     welcome_message_sent: 'Welcome Message Sent',
     record_automatically_closed: 'Record Automatically Closed',
-    monitoring_complete_message_sent: 'Monitoring Complete Message Sent'
+    monitoring_complete_message_sent: 'Monitoring Complete Message Sent',
+    assessment_email_error: 'Assessment Email Error'
```

Comment:
do we want to convey that this was not their fault, and was a fault with the system?

Reply:
I have a thread out in the UI channel finalizing the wording for this so I'll defer to UAT/PO feedback.

Reply:
Updated the language in https://github.com/SaraAlert/SaraAlert/pull/990/commits/10873dceff43669a3b0518853e43e80ccf445076